### PR TITLE
RUN-1346: remove Job and Report filters saved in DB

### PR DESCRIPTION
**Pull request by gschueler**
**Is this a bugfix, or an enhancement? Please describe.**
 Removes DB-based storage for "JobFilters" and "ReportFilters"

This PR extends https://github.com/rundeck/rundeck/pull/8558

**Describe the solution you've implemented**

- [x] Removes DB-based storage for "JobFilters" and "ReportFilters"
- [x] Updates GUI for Activity list to use browser local storage for user filters

**Additional context**
Job filters have not been migrated to browser storage

### CODE REVIEW

File: rundeckapp/grails-spa/packages/ui-trellis/src/library/stores/ActivityFilterStore.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  [key: string]: ProjectActivityFilters
+}
+
+/**
+ * Use local storage for managing saved node filters
+ */
+export class ActivityFilterStore {
+
+  storeFilters(filters: StoredActivityFilterSet) {
+    debugger
```
Comment (JoseOrtiz) [inline rundeckapp/grails-spa/packages/ui-trellis/src/library/stores/ActivityFilterStore.ts:23]: Is this `debugger` intentional?

File: rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/savedFilters.vue [hunk 2]
```diff
@@ -93,47 +78,16 @@ export default defineComponent({
...
-        }
-        if (response.parsedBody && response.parsedBody.success) {
-          const newfilter=Object.assign({ name: name }, this.query)
-          this.filters.push(newfilter);
-          this.$emit("select_filter", newfilter);
-        }
-      } catch (error) {
-        this.loadError = error.message;
-      }
+      debugger
```
Comment (JoseOrtiz) [inline rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/savedFilters.vue:85]: Is this `debugger` intentional?

Comment (gschueler) [reply rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/savedFilters.vue:85]: no, i'll remove them

Comment (gschueler) [reply rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/savedFilters.vue:85]:
```suggestion
```

File: rundeckapp/grails-spa/packages/ui-trellis/src/library/stores/ActivityFilterStore.ts (hunk 1 above)
Comment (gschueler) [reply rundeckapp/grails-spa/packages/ui-trellis/src/library/stores/ActivityFilterStore.ts:23]:
```suggestion
```

**Pull request merged**